simulations.py

```diff
@@ -65,41 +77,62 @@ def inv_nla_jit(A):
   return np.linalg.inv(A)
 
 def update_Ui(Cit,Ui,mu_Ui, Sigma_Ui, Nset):
+    """ Update beliefs using Baysian updating on Normal Distribution
+    """ 
+    
+    # https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Conditional_distributions
+    # μ_bar = μ_1 + Ε12 Ε22^-1 ( a - μ_2 )  
+
     x1 = Cit
     x2 = [n for n in Nset if n not in Cit]
     Nit = [n for n in Nset if n not in Cit]
+
     mu1 = np.array([mu_Ui[n] for n in x1]).reshape((1,len(x1)))
     mu2 = np.array([mu_Ui[n] for n in x2]).reshape((1,len(x2)))
+
     Sigma11 = np.ones((len(x1),len(x1)))
     Sigma21 = np.ones((len(x2),len(x1)))
+
     for i in range(len(Cit)):
         for j in range(len(Cit)):
-            Sigma11[i,j] = Sigma_Ui[Cit[i],Cit[j]]
+            Sigma11[i,j] = Sigma_Ui[Cit[i],Cit[j]] 
+
     for i in range(len(Cit)):
         for j in range(len(Nit)):
-            Sigma21[j,i] = Sigma_Ui[Cit[i], Nit[j]]
+            Sigma21[j,i] = Sigma_Ui[Cit[i], Nit[j]] 
+
     a = np.array([Ui[n] for n in x1]).reshape((1,len(x1)))
     inv_mat = inv_nla_jit(Sigma11)
     inner = np.matmul(Sigma21, inv_mat)
     mubar = mu2 + (np.matmul(inner,(a-mu1).T)).T
     mu_new = Ui
+
     for i in range(len(x2)):
         mu_new[x2[i]] = mubar[0,i]
     return mu_new
 
 ## CHOICE FUNCTIONS
 def choice_helper(Cit,mu, Nset):
```

Comment:
can you rename the mu so that this is a bit less confusing? This is no longer taking in the means, but rather the certainty equivalents